engine/engine/content/builtins/manifests/ios/Info.plist

```diff
@@ -47,7 +47,7 @@
         <key>CFBundleInfoDictionaryVersion</key>
         <string>6.0</string>
         <key>CFBundleName</key>
-        <string>dmengine</string>
+        <string>{{bundle-name}}</string>
```

Comment:
It's a bit weird that we need it at all, since the docs say it's only used if the CFBundleDisplayName isn't set (which is is)

https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundlename